stablehlo/dialect/TypeInference.h

```diff
@@ -231,8 +231,8 @@ LogicalResult inferGatherOp(
     SmallVectorImpl<ShapedTypeComponents>& inferredReturnShapes);
 
 LogicalResult inferGetDimensionSizeOp(
-    MLIRContext* context, std::optional<Location> location,
-    SmallVectorImpl<Type>& inferredReturnTypes);
+    MLIRContext* context, std::optional<Location> location, Type operandType,
```

Comment:
Now that we're passing in `operandType`, we can drop the `context` parameter and obtain it from the type - `operandType.getContext()`.

Reply:
Sure